app/event/eventController.js

```diff
@@ -62,6 +86,18 @@
             return numberOfChar >= LIMIT_CHARACTERS;
         };
 
+        eventCtrl.isAuthorized = function isAuthorized(event) {
```

Comment:
canDelete, is authorized é muito genérico, e a implementação é específica p remover